.github/workflows/build-wsl.yaml

```diff
@@ -286,3 +288,29 @@ jobs:
           git add .
           git commit -m "Auto-update build info"
           git push origin master
+
+      - name: Checkout WSL repo
```

Comment:
Maybe add the `if: ${{ steps.modified-artifacts.outputs.needs-wiki-update == 'true' }}` in there and do the 2 clones in a single step, as both is to get the code to notify the tracker.